tests/admin_widgets/false_timezone_warning_templates/admin/base.html

```diff
@@ -0,0 +1,88 @@
+{% load i18n admin_static %}<!DOCTYPE html>
```

Comment:
A very simple test would be a plain JavaScript test that ensures that `DateTimeShortcuts.timezoneOffset` is still `0` after running `DateTimeShortcuts.init()`. [This property is checked to determine if the timezone warnings should be shown](https://github.com/django/django/blob/master/django/contrib/admin/static/admin/js/admin/DateTimeShortcuts.js#L62). This could be done in [js_tests/admin/DateTimeShortcuts.test.js](https://github.com/django/django/blob/master/js_tests/admin/DateTimeShortcuts.test.js) rather easy. Its not really a test if the incorrect warnings are shown, but at least it tests the condition for the timezone warnings. This will also probably cause false positives if the JS test client is running in UTC, unless I find a way to reliably mock this.